### PR TITLE
broken URL fixes Nov 2024

### PR DESCRIPTION
**Pull request by gufmar**
- clean up obsolete broken URLs
- fix the raw .jsonld file link in the governance action example

Bonus: added the `announcementBar` switch in the `docusaurus.config.js` as just implemented on cardano.org

### CODE REVIEW

File: docs/get-started/cardano-cli/governance/governance-actions.md
Comment (gufmar) [inline docs/get-started/cardano-cli/governance/governance-actions.md:1]: @gitmachtl or @disassembler 
may I ask you to quickly check if this replacement of the URL is ok for the code examples? 
Imo it still points to the same jsondld content, and it's just about to fix the link to it.

**Review by rphair — APPROVED** [review]
@gufmar I reviewed it to my own satisfaction yesterday but was waiting for a response to https://github.com/cardano-foundation/developer-portal/pull/1353#discussion_r1839953348 ... with no objection yet I believe this is good to merge as-is.

**Review by katomm — APPROVED** [review]

File: docs/get-started/cardano-cli/governance/governance-actions.md
Comment (gufmar) [reply docs/get-started/cardano-cli/governance/governance-actions.md:1]: discussed and solved via chat

**Review by weqanhet — APPROVED** [review]

**Comment by weqanhet** [comment]
LGTM

**Pull request merged**